com.unity.render-pipelines.high-definition/Editor/BuildProcessors/HDRPPreprocessShaders.cs

```diff
@@ -287,14 +287,14 @@ internal bool StripShader(HDRenderPipelineAsset hdAsset, ComputeShader shader, s
             if (inputData.shaderKeywordSet.IsEnabled(m_ScreenSpaceShadowOFFKeywords) && shadowInitParams.supportScreenSpaceShadows)
                 return true;
 
-            if (inputData.shaderKeywordSet.IsEnabled(m_ScreenSpaceShadowONKeywords) && !shadowInitParams.supportScreenSpaceShadows)
-                return true;
-
-            if (inputData.shaderKeywordSet.IsEnabled(m_MSAA) && !hdAsset.currentPlatformRenderPipelineSettings.supportMSAA)
+            if (inputData.shaderKeywordSet.IsEnabled(m_MSAA) && (hdAsset.currentPlatformRenderPipelineSettings.supportedLitShaderMode == RenderPipelineSettings.SupportedLitShaderMode.DeferredOnly))
```

Comment:
Both this and https://github.com/Unity-Technologies/Graphics/pull/3840/files#diff-2e328001603088c74758f8d9ccc9157aae3f8c220f0c0786af92db676ebad710R80 will be stripped less often now, do we have data how many are we not stripping now with forward?